FunctionalTableData/CellStyle.swift

```diff
@@ -116,13 +130,15 @@ extension CellStyle: Equatable {
 	public static func ==(lhs: CellStyle, rhs: CellStyle) -> Bool {
 		var equality = lhs.bottomSeparator == rhs.bottomSeparator
 		equality = equality && lhs.topSeparator == rhs.topSeparator
+		equality = equality && lhs.separatorColor == rhs.separatorColor
 		equality = equality && lhs.highlight == rhs.highlight
 		equality = equality && lhs.accessoryType == rhs.accessoryType
 		equality = equality && lhs.selectionColor == rhs.selectionColor
 		equality = equality && lhs.backgroundColor == rhs.backgroundColor
 		equality = equality && lhs.backgroundView == rhs.backgroundView
```

Comment:
This view comparison should probably use `isEqual` instead. AFAIK that defaults to pointer equality for `UIView`s anyway, but consumers of the API may want to avoid an unnecessary style update for custom views they know are functionally identical.

Reply:
== should be doing isEqual for NSObjects, === does pointer comparisons

Reply:
TIL! 😮